lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -1982,4 +1982,13 @@ public void deleteLogForLessonsItem(SimplePageItem item) {
 			log.warn("Failed to delete lessons log for item {}", item.getId());
 		}
 	}
+
+	public void deleteCommentsForLessonsItem(SimplePageItem item) {
+		try {
+			DetachedCriteria d = DetachedCriteria.forClass(SimplePageComment.class).add(Restrictions.eq("itemId",item.getId()));
+			getHibernateTemplate().deleteAll(getHibernateTemplate().findByCriteria(d));
+		} catch (DataAccessException e) {
+			log.warn("Failed to delete SimplePageComments for item {}", item.getId());
```

Comment:
I would make this an error, and add e.toString() so we get the full information in the log